dev/com.ibm.ws.rest.handler.config_fat/fat/src/com/ibm/ws/rest/handler/config/fat/ConfigRESTHandlerTest.java

```diff
@@ -452,62 +478,6 @@ public void testConfigDataSource() throws Exception {
         assertEquals(err, true, j.getBoolean("transactional"));
         assertNotNull(err, j = j.getJsonObject("properties.derby.embedded"));
         assertEquals(err, "memory:doesNotExist", j.getString("databaseName"));
-
-        j = json.getJsonObject(5);
-        assertEquals(err, "dataSource", j.getString("configElementName"));
-        assertEquals(err, "transaction/dataSource[default-0]", j.getString("uid"));
```

Comment:
This is our only test of config output for a dataSource nested under a singleton pid.  Do not remove it.